src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -755,7 +781,12 @@ export class WebchatUI extends React.PureComponent<
 			);
 
 		const handleOnClose = () => {
-			onClose?.();
+			if (showInformationMessage) {
+				onClose?.();
+			} else {
+				onSetShowPrevConversations(false);
+				onSetShowHomeScreen(true);
+			}
```

Comment:
Close button should always close the widget. To get back to home screen, there is a back button.

There might be a case where it's not always true (e.g. some pop-up), here though it is not the case.

Reply:
fixed